### ISSUE TITLE
b2 jamfile syntax warning

### ISSUE THREAD

**Issue by jeking3** [boostorg/multiprecision#93]
```
../libs/multiprecision/test/Jamfile.v2:58: Unescaped special character in argument <toolset>msvc:<cxxflags>/fp:precise
```

**Comment by NAThompson** [comment]
Do you know which character is regarded as special in this? I get this warning all the time and have no clue how to fix it.

**Comment by NAThompson** [comment]
A colon sets a low bar for a special character. In any case, it's fixed in  [30bc417](https://github.com/boostorg/multiprecision/commit/30bc41706d8e8a1cb7e7f060c075123acb4a164c).

**Comment by jeking3** [comment]
Thanks for tackling that so quickly.
